content/applications/general/iot/connect.rst

```diff
@@ -0,0 +1,153 @@
+=============================
+IoT system connection to Odoo
+=============================
+
+Prerequisites
+=============
+
+To connect the IoT system to an Odoo database, the following prerequisites must be met:
+
+- The Odoo database must be a **production** instance.
```

Comment:
in practice it is untrue, but indeed they might run into issues with duplicate/staging (especially with HTTPS)